src/state/migrations/index.ts

```diff
@@ -7,4 +7,5 @@ export const migrations = {
   1: clearOpportunities,
   2: clearPortfolio,
   3: clearOpportunities,
+  4: clearOpportunities,
```

Comment:
see comment above line 4 🙂
`// when adding a migration, dont forget to increment `persistConfig.version` in `src/state/store.js``

Reply:
hell yes